src/locale/pages/organizeCallAssignment/en.yml

```diff
@@ -1,6 +1,9 @@
 blocked:
   callBackLater: Asked us to call back later
   calledTooRecently: Called too recently
+  cooldownHelperText: Set the minimum time between attempts to reach a target
+  cooldownLabel: Cooldown time in hours
+  hours: '{cooldown, plural, =0 {None} =1 {One hour} other {# hours}}'
```

Comment:
I know I was the one to write this, but then I saw "One hour" in the interface and didn't like it. Maybe we should use "1 hour" instead?

Reply:
Agreed, changing :D